californium-core/src/main/java/org/eclipse/californium/core/CoapServer.java

```diff
@@ -147,7 +147,7 @@ public CoapServer(final NetworkConfig config, final int... ports) {
 		}
 		
 		// resources
-		this.root = createRoot();
+		this.root = new RootResource();
```

Comment:
This effectively removes the possibility to use another root resource by extending `CoapServer`.